code/modules/zombie/organs.dm

```diff
@@ -1,22 +1,21 @@
-#define START_TIMER reanimation_timer = world.time + rand(600,1200)
-
 /obj/item/organ/internal/zombie_infection
 	name = "festering ooze"
 	desc = "A black web of pus and viscera."
 	parent_organ = "head"
 	slot = "zombie_infection"
 	icon_state = "blacktumor"
 	var/causes_damage = TRUE
-	var/reanimation_timer
-	var/datum/species/old_species = new /datum/species/human
-	var/living_transformation_time = 5
-	var/converts_living = FALSE
+	var/datum/species/old_species = /datum/species/human
+	var/living_transformation_time = 3
```

Comment:
Why 3 instead of 30? Is this a testing variable?

This is bad since the zombie will revive 3 seconds (or less) after getting killed. It seems a bit inconsistent to me, but happened 4 - 5 time when I tested shooting a SAW at them.

Reply:
This isn’t the revive time variable, it’s how long they’re stunned for and do their jitter animation when converted